File: power of the number.go
package main

import ("fmt"
        "math"
)

func main() {
   var num int
   fmt.Scanf("%d", &num)
   i := num * num
   f := float64(num)
   j := math.Pow(f, 2)
   fmt.Println(i)
   fmt.Printf("%d power of 2 is %f \n", num, j)
   fmt.Printf("%d power of 2 is %d \n", num, int(j))
}

File: src/powerofthenumber/powerofthenumber.java
package powerofthenumber;
import java.util.Scanner;


public class powerofthenumber {
	    public static void main(String [] args) {
	    	Scanner in = new Scanner(System.in);
	    	int num = in.nextInt();
	    	System.out.println(num + " to the power of 2 " + (int)Math.pow(num, 2));
	    	in.close();
	    	       //System.out.println("Hello World"); // prints Hello World)
	    }

}
